ext/civicrm_admin_ui/README.md

```diff
@@ -1,3 +1,5 @@
 # civicrm_admin_ui
 
 Enable this extension for some screens under "Administer CiviCRM" to be configurable SearchKit displays.
+
+It does wonderful stuff
```

Comment:
What is the proper termination of this sentence?

Reply:
Hello

Reply:
Hi Tim